### ISSUE TITLE
Filter with stock symbol such as $aapl or $fb doesnt seem to return any streamed tweets

### ISSUE THREAD

**Issue by thomasswede** [fennb/phirehose#94]
Cant seem to get the filtering to track words with the $ symbol in them. Tracking AAPL works, but not $AAPL. Doing a regular twitter api search with $aapl works as expected, but not via streaming api and using Phirehose. Any hints or tips?

**Comment by DarrenCook** [comment]
>  Tracking AAPL works, but not $AAPL. 

Are you using single quotes, or double quotes, around $AAPL in your PHP
script? Use single quotes, or escape the $ by putting a \ before it if
using double-quotes.

Darren

P.S. Then it should work, according to the Twitter API docs:
 https://dev.twitter.com/streaming/overview/request-parameters#track

**Comment by thomasswede** [comment]
Yes, Darren, have tried all of your suggestions, and it still doesnt generate any incoming streaming tweets. Even tried with %24 instead of the $ etc. The URL that Phirehose generates to connect to the twitter streaming api is properly formatted when debugging ( it says .... %24aapl%2c%24fb and so on. Really strange this is.
